exercises/practice/minesweeper/.meta/Example.roc

```diff
@@ -0,0 +1,45 @@
+module [annotate]
+
+isBomb : List Str, I64, I64 -> Result Bool [OutOfBounds]
+isBomb = \rows, nx, ny ->
+    if nx < 0 || ny < 0 then
+        Err OutOfBounds
+    else
+        rowChars = rows |> List.get? (Num.toU64 ny) |> Str.toUtf8
+        rowChars |> List.get? (Num.toU64 nx) |> Bool.isEq '*' |> Ok
+
+countNeighbors : List Str, U64, U64 -> Num *
+countNeighbors = \rows, x, y ->
+    [-1, 0, 1]
+    |> List.map \dy ->
+        [-1, 0, 1]
+        |> List.map \dx ->
+            when isBomb rows (Num.toI64 x + dx) (Num.toI64 y + dy) is
+                Ok bomb -> if bomb then 1 else 0
+                Err OutOfBounds -> 0
+        |> List.sum
+    |> List.sum
+
+annotate : Str -> Str
+annotate = \minefield ->
```

Comment:
I think it would be worth completely converting the input to a `List (List U8)` up front so that there don't have to be other unrelated `Str.toUtf8` calls in the body of the other functions.

Reply:
Good point, I'll do that.